### ISSUE TITLE
v0.60.2 (Api) Json.parse error while bunding graphql function

### ISSUE THREAD

**Issue by dkershner6** [serverless-stack/serverless-stack#1280]
```
> sst build --stage staging

Using stage: staging
Preparing your SST app
Synthesizing CDK
Building function src/api/health/get.default
Building function src/api/graphql/index.default

SyntaxError: /home/dkershner/repos/resonance-api-supply/.sst/artifacts/d452263f/package.json: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Object.readFileSync (/home/dkershner/repos/resonance-api-supply/node_modules/jsonfile/index.js:52:17)
    at installNodeModules (/home/dkershner/repos/resonance-api-supply/node_modules/@serverless-stack/core/dist/runtime/handler/node.js:296:41)
    at Object.bundle (/home/dkershner/repos/resonance-api-supply/node_modules/@serverless-stack/core/dist/runtime/handler/node.js:217:13)
    at Object.bundle (/home/dkershner/repos/resonance-api-supply/node_modules/@serverless-stack/core/dist/runtime/handler/handler.js:19:16)
    at new Function (/home/dkershner/repos/resonance-api-supply/node_modules/@serverless-stack/resources/src/Function.ts:361:39)
    at SupplyApiStack.createGraphQLRoutes (/home/dkershner/repos/resonance-api-supply/lib/SupplyApiStack.ts:401:33)
    at new SupplyApiStack (/home/dkershner/repos/resonance-api-supply/lib/SupplyApiStack.ts:97:14)
    at Object.ResonanceApiSupply (/home/dkershner/repos/resonance-api-supply/lib/index.ts:13:5)
    at Object.<anonymous> (/home/dkershner/repos/resonance-api-supply/.build/run.js:94:16)

There was an error synthesizing your app.
```

The package.json in question is a blank file.

This is confirmed working on `0.59.1`, but breaks on latest (`0.60.2`).

This is a very large graphql function with many resolvers running `apollo-server-lambda`

**Comment by dkershner6** [comment]
This does appear related to GraphQL/Apollo somehow, I was able to upgrade a fully REST API no problem.

**Comment by dkershner6** [comment]
I was able to build a separate GraphQL/Apollo API with no issues, so there is something else unique here...

**Comment by thdxr** [comment]
This should be fixed in 0.60.4

**Comment by dkershner6** [comment]
@thdxr , CONFIRMED!

Thanks.
